scripts/release/asset_publish.py

```diff
@@ -402,6 +405,49 @@ def create_asset(
         failure_list.append(asset)
 
 
+def update_asset_metadata(asset: AssetConfig, registry_name: str):
+    """Update the mutable metadata of asset."""
+    if asset.type == assets.AssetType.MODEL:
+        model_name = asset.name
+        model_version = asset.version
+        spec_path = asset.spec_with_path
+        model_config = asset.extra_config_as_object()
+
+        tags, description = None, None
+
+        # get tags to update from model spec file
+        try:
+            with open(spec_path) as f:
+                model_spec = YAML().load(f)
+                if "tags" in model_spec:
+                    tags = {"replace": model_spec["tags"]}
+        except Exception as e:
+            logger.log_error(f"Failed to get tags for model {model_name}: {e}")
+
+        # get updated description for model
+        try:
+            model_description_file_path = Path(spec_path).parent / model_config.description
+            logger.print(f"model_description_file_path {model_description_file_path}")
+            if os.path.exists(model_description_file_path):
+                with open(model_description_file_path) as f:
+                    description = f.read()
+        except Exception as e:
+            logger.log_error(f"Failed to get description for model {model_name}: {e}")
+
+        update = AssetVersionUpdate(versions=[model_version],
+                                    tags=tags,
```

Comment:
Do we update even if loading of tags/description fail?

Reply:
Could you please add UTs to cover such cases?

Reply:
No, if tags,description won't load it will get set as None, and for none type values wont get affected. It will only update if and only if it get valid description and tag